lint/highlight.py

```diff
@@ -400,6 +433,28 @@ def draw(self, view):
                 region = sublime.Region(self.newlines[line], self.newlines[line])
                 gutter_regions[error_type].append(region)
 
+        def hide_phantoms(href):
+            view.erase_phantoms("sublimelinter")
+
+        for _line, _error in self.errors.items():
+            _column = _error[0][0]
```

Comment:
Is it in line of current style to prefix local variables with underscore? It doesn't seem like it.